Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealitySpatialMeshObserver.cs

```diff
@@ -175,6 +175,13 @@ public override void Initialize()
                     UnityEditor.PlayerSettings.WSACapability.SpatialPerception,
                     this.GetType());
 #endif
+            // If we aren't using a HoloLens or there isn't an XR device present, return.
+            if (observer == null || HolographicSettings.IsDisplayOpaque || !XRDevice.isPresent) { return; }
+
+            if (RuntimeSpatialMeshPrefab != null)
+            {
+                AddRuntimeSpatialMeshPrefabToHierarchy();
```

Comment:
@CDiaz-MS, do you recall why these calls were put into the platform-specific observers instead of the base layer where the method and prefab handling themselves live?

Reply:
@keveleigh I recall it was because there were platform specific conditions that needed to be checked before the prefab is inserted into the hierarchy.  Although it does make more sense to add this to the BaseSpatialMeshObserver Initialize, I'll do some on device testing to confirm.